src/Symfony/Component/Security/Core/Tests/Validator/Constraints/UserPasswordValidatorTest.php

```diff
@@ -95,9 +95,7 @@ public function provideConstraints(): iterable
     {
         yield 'Doctrine style' => [new UserPassword(['message' => 'myMessage'])];
 
-        if (\PHP_VERSION_ID >= 80000) {
-            yield 'named arguments' => [eval('return new \Symfony\Component\Security\Core\Validator\Constraints\UserPassword(message: "myMessage");')];
-        }
+        yield 'named arguments' => [eval('return new \Symfony\Component\Security\Core\Validator\Constraints\UserPassword(message: "myMessage");')];
```

Comment:
```suggestion
        yield 'named arguments' => [new UserPassword(message: 'myMessage')];
```

We should remove the `eval()`s everywhere. They were making sure we don't get parse errors on PHP 7.